docs/_includes/arguments/table-card-group.md

```diff
@@ -0,0 +1,30 @@
+### CardsData
+
+| Name  | Type   | Required | Description                                       |
+| ----- | ------ | -------- | ------------------------------------------------- |
+| title | String | Yes      | The text to be displayed as the section title     |
+| icon  | String | Yes      | The name of the icon to use in the section header |
+| id    | String | Yes      | The Unique ID                                     |
+
+### CardsData
```

Comment:
There's two `CardsData` sections here, and it looks like no relationship between them?